src/main.js

```diff
@@ -252,6 +252,11 @@ function configureCommandlineSwitchesSync(cliArgs) {
 	 */
 	app.commandLine.appendSwitch('disable-features', 'CalculateNativeWinOcclusion');
 
+	/**
+	 * Allow media to play even if no user gesture is used to fix issues like #176284
+	 */
+	app.commandLine.appendSwitch('autoplay-policy', 'no-user-gesture-required');
```

Comment:
If this is restricted to just the renderer (not webviews) then it's not a big deal imo, it will not work in web of course though.